dev-cpp/gtest-parallel/gtest-parallel-20220621.ebuild

```diff
@@ -0,0 +1,34 @@
+# Copyright 2025 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+PYTHON_COMPAT=( python3_{13..14} )
+inherit python-r1
+
+DESCRIPTION="Run Google Test suites in parallel."
```

Comment:
wrt category, just because gtest is in dev-cpp, I don't think this belongs there as well -- dev-util would fit better I think.

Reply:
In this case I picked dev-cpp as its tied to gtest which is also there, and  as such gtest-parallel is also going to be used primarily by C++ projects.

Reply:
I could go either way but I guess dev-util makes more sense as it's not a library. But dev-util is also pretty big while dev-cpp is much smaller.